_data/changelogs/docs-getting-started-devs-phase-2.yml

```diff
@@ -2,6 +2,16 @@ title: "Phase 2: Compile"
 type: documentation
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Added documentation for `settings.compile.browserslist` and `settings.compile.sassSourcemaps`.
+    affectsGuidance: true
+    githubPr: 2750
+    githubRepo: uswds-site
+  - date: NNNN-NN-NN
+    summary: Replaced "Gulp 4" references with "Gulp 5".
```

Comment:
**suggestion:** If we remove the gulp version from the documentation we should update this summary as well

Reply:
Good flag! Updated the summary here to "Removed the version number from "Gulp 4" references." in [4426600](https://github.com/uswds/uswds-site/pull/2750/commits/44266000175d05ec29501f17b4187efab1b15f53)